src/pages/LogIn/Login.js

```diff
@@ -2,8 +2,100 @@ import React, { Component } from 'react';
 import './Login.scss';
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  postLogin = () => {
+    const { id, pw } = this.state;
+    if (
+      this.state.id.includes('@') &&
+      this.state.id.includes('.com') &&
+      this.state.pw.length >= 8
+    ) {
+      fetch('http://172.30.1.25:8000/users/signin', {
```

Comment:
```suggestion
      fetch(`${API}/users/signin`, {
```
ip 주소는 빈번히 바뀌고 여러 곳에서 사용되기 때문에 src/config.js 등의 경로에 분리해두는 것이 좋습니다

Reply:
넵 수정완료했습니다 ~!